generators/entity/templates/src/main/webapp/app/entities/infinite-scroll-template.ejs

```diff
@@ -29,7 +29,11 @@
         <%_ } else { _%>
             if (vm.currentSearch) {
```

Comment:
Using `searchQuery` here was a mistake, it should be `currentSearch`. I think `currentSearch` is used because without it, if a user changes the text in the search box without performing a search, the search will be performed any time the user sorts the current results or changes pages.